### PR TITLE
ACI Intf Policy Port Security: Update module to use new URL method

### PR DESCRIPTION
**Pull request by jmcgill298**

### CODE REVIEW

**Comment by jmcgill298** [comment]
trailing comma for @dagwieers :)

**Comment by dagwieers** [comment]
Mr dag wieers is pleased !

File: library/aci_intf_policy_port_security.py [hunk 1]
```diff
@@ -80,32 +80,27 @@ def main():
...
+        required_if=[['state', 'absent', ['port_security']],
+                     ['state', 'present', ['port_security']]]
     )
 
     port_security = module.params['port_security']
     description = module.params['description']
-    # TODO: validate that max_end_points is in the acceptable range
     max_end_points = module.params['max_end_points']
+    if max_end_points is not None and max_end_points not in range(12001):
+        module.fail_json(msg='The "max_end_points must be between 0 and 12000')
```
Comment (dagwieers) [inline library/aci_intf_policy_port_security.py]: I suspect you adding that typo just to see if I am watching carefully. I SAW IT !!

File: library/aci_intf_policy_port_security.py [hunk 2]
```diff
@@ -80,32 +80,27 @@ def main():
     module = AnsibleModule(
         argument_spec=argument_spec,
         supports_check_mode=True,
+        required_if=[['state', 'absent', ['port_security']],
+                     ['state', 'present', ['port_security']]]
```
Comment (dagwieers) [inline library/aci_intf_policy_port_security.py]: Trailing comma ! (although I agree this is a weird one ;-))

File: library/aci_intf_policy_port_security.py (hunk 1 above)
Comment (jmcgill298) [inline library/aci_intf_policy_port_security.py]: good catch, will fix shortly

**Comment by jmcgill298** [comment]
Fixed

**Pull request merged**